common/src/main/resources/ts/models/FormElement/Question.ts

```diff
@@ -50,14 +50,28 @@ export class Question extends FormElement {
         this.choices = new QuestionChoices();
         this.children = new Questions();
         this.placeholder = null;
-        this.cursor_min_val = null;
-        this.cursor_max_val = null;
-        this.cursor_step = null;
-        this.cursor_min_label = null;
-        this.cursor_max_label = null;
+        this.specific_fields = new QuestionSpecificFields(this.id);
         this.form_element_type = FormElementType.QUESTION;
     }
 
+    build(data: IQuestionResponse) : Question {
+        super.build(data);
+        this.question_type = data.questionType ? data.questionType : null;
+        this.statement = data.statement ? data.statement : null;
+        this.mandatory = data.mandatory ? data.mandatory : false;
```

Comment:
```suggestion
        this.mandatory =  data.mandatory || false;
```
Pour la science, je crois que ça, ça fonctionne